modules/experimental/src/gltf/gltf-material-parser.js

```diff
@@ -67,14 +67,30 @@ export default class GLTFMaterialParser {
         gltfTexture.texture.sampler.parameters) ||
       {};
 
+    const image = gltfTexture.texture.source.image;
+    let textureOptions;
+    let specialTextureParameters = {};
+    if (image.compressed) {
```

Comment:
Hmm, why do we need this logic? Add a comment?

Is this always needed for compressed textures? If we have a standardized format for compressed textures, can this just be handled in the Texture constructor so that we can pass in the texture and not have to handle this here?

Reply:
1. Data structure we pass to Texture2D constructor is different:
  - For ImageBitmap we need do pass `{data: image}`;
  - For compressed texture we already have object with data field `{data: Buffer, compressed: true, mipmaps: false, ...}`;
2. We add important parameter: `TEXTURE_MIN_FILTER` . It has different value for compressed textures